holoviews/plotting/bokeh/element.py

```diff
@@ -49,7 +49,7 @@
 
 class ElementPlot(BokehPlot, GenericElementPlot):
 
-    bgcolor = param.Parameter(default='white', doc="""
+    bgcolor = param.Parameter(default=None, allow_None=True, doc="""
```

Comment:
``default=None`` implies ``allow_None`` so ``allow_None`` is superfluous here.

    As a special case, if allow_None=True (which is true by default if
    the parameter has a default of None when declared) then a value
    of None is also allowed.

Reply:
Don't mind though tbh.

Reply:
I don't really mind either way but I am in the habit of explicitly setting ``allow_None``. I'll get rid of it if you ask me again.